### PR TITLE
Double search

### PR DESCRIPTION
**Pull request by dnoneill**
Screenshot 2024-08-28 at 5 08 13 PM

### CODE REVIEW

**Comment by jcoyne** [comment]
This seems like an unusual and unexpected way of customizing blacklight. Have you considered setting the search_bar_component class instead? (overriding Blacklight::SearchBarComponent)?

**Comment by dnoneill** [comment]
Done.

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -40,6 +40,7 @@ class CatalogController < ApplicationController
 
     config.index.document_component = Earthworks::SearchResultComponent
     config.index.title_field = Settings.FIELDS.TITLE
+    config.index.search_bar_component = Earthworks::SearchBarComponent
```
Comment (jcoyne) [inline app/controllers/catalog_controller.rb:43]: Do we want to use this component for the show view too?

Comment (dnoneill) [reply app/controllers/catalog_controller.rb:43]: I don't see a config.show.search_header_component in blacklight and it looks like the show page is using the custom search_header_component

**Review by jcoyne — APPROVED** [review]

**Pull request merged**